files/build_templates/pmon.service.j2

```diff
@@ -6,9 +6,14 @@ Before=ntp-config.service
 
 [Service]
 User={{ sonicadmin_user }}
+{% if disable_pmon == 'y' %}
```

Comment:
it is better to add some hook to say if onie string matches a specific string, then we do not start pmon docker.

Reply:
@jleveque to give any comments on this.

Reply:
Agree with Guohan that it might be easier to maintain if we test for ONIE platform strings here, instead of introducing a new variable.